fledging.qmd

```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
 and want to continue their adventures in cloud computing, have the knowledge
 and resources to develop their own solutions.
```

Comment:
The idea of "fledging" and how to support users is a work-in-progress developed with the NASA Openscapes Mentor community, and this webpage will be periodically updated to reflect current thinking and experiences.

Reply:
Thanks @jules32 - sorry I didn't see where your suggested text is?